src/tests/ftest/cart/no_pmix/cart_nopmix_multictx_one_node.py

```diff
@@ -51,6 +51,10 @@ def setUp(self):
         ofi_ctx_num = self.params.get("CRT_CTX_NUM", '/run/defaultENV/')
         ofi_share_addr = self.params.get("CRT_CTX_SHARE_ADDR",
                                          '/run/defaultENV/')
+    def tearDown(self):
+        """ Tear down """
+        super(TestWithoutServers, self).tearDown()
```

Comment:
(pylint-bad-super-call) Bad first argument 'TestWithoutServers' given to super()